Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM wisvch/debian:stretch
+MAINTAINER Mark Janssen <mark@praseodym.net>
+
+RUN apt-get update && apt-get install -y nodejs nodejs-legacy && rm -rf /var/lib/apt/lists/*
+COPY florriebot.tar.gz /tmp
```

Comment:
It would mean downloading 200-300MB of dependencies, installing and then removing them again, so I prefer this approach.